config/v2/config.go

```diff
@@ -124,6 +124,7 @@ type Providers struct {
 	Okta      *OktaProvider      `yaml:"okta,omitempty"`
 	Snowflake *SnowflakeProvider `yaml:"snowflake,omitempty"`
 	Datadog   *DatadogProvider   `yaml:"datadog,omitempty"`
+	Sentry    *SentryProvider    `yaml:"sentry,omitempty"`
```

Comment:
Added to support tfe-infra repo.

Now that we need to put providers in `terraform { required_providers { ... } }` *and* you can only have one such block, we will need to add all providers we want to use here :(.